chsdi/templates/htmlpopup/atlas_basisgebiete.mako

```diff
@@ -1,6 +1,7 @@
 <%inherit file="base.mako"/>
 
 <%def name="table_body(c, lang)">
+<% c['stable_id'] = True %>
```

Comment:
In general: also add an `or '-'` clause in the below tables (as done in the other atlas_* makos)?